src/main/java/interpol/entity/Crime.java

```diff
@@ -17,6 +18,9 @@ public class Crime {
     @Column (name = "punishment")
     private String punishment;
 
+    @ManyToMany(fetch = FetchType.EAGER, mappedBy = "crimes")
```

Comment:
It's better to avoid using ManyToMany -> instead of this, please, create anoter table for mapping of this two tables.
And general question: do we expect such relation here?

Reply:
I think there should be such connection (ManyToMany between "Crime" and "OrderDetail") because some time ago table "OrderDetail" had name "Criminal" and we decided that one criminal can have many crimes, and one crime can be conducted by many criminals.

Reply:
I mean, that we should clearly create 3rd table, which will be a mapping table, and in java code this table will have 2 ManyToOne relations

Reply:
I see. Thank you.